python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -80,6 +80,15 @@ def divide(attrs, inputs, proto_obj):
         return op_value, new_attr, inputs
     return 'broadcast_div', new_attr, inputs
 
+def mean(attrs, inputs, proto_obj):
+    """Mean of two input tensors."""
+    concat_input = []
+    for op_input in inputs:
+        concat_input.append(symbol.expand_dims(op_input, axis=0))
+    concat_sym = symbol.concat(*concat_input, dim=0)
+    mean_sym = symbol.mean(concat_sym, axis=0)
```

Comment:
this is mean on axis 0, are you sure it is the desired behavior?

Reply:
yes, i am this is correct. We are doing an unsqueeze along axis=0, hence the mean along axis=0. It has also passed all the operator tests.